README.md

```diff
@@ -1,7 +1,7 @@
 # jv-oop
 
-- Создать класс Machine с методами _public abstract void doWork()_ и _public abstract void stopWork()_. 
-- Создать классы Track, Bulldozer, Excavator которые унаследуют Machine. 
+- Создать класс core.basesyntax.Machine с методами _public abstract void doWork()_ и _public abstract void stopWork()_.
+- Создать классы Track, Bulldozer, Excavator которые унаследуют core.basesyntax.Machine.
```

Comment:
Don't need changes in description. We don't need the fully-qualified name of class there.